examples/compose.js

```diff
@@ -0,0 +1,29 @@
+import dotty from 'dotty';
+import merge from 'lodash/object/merge';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return composables.map(composable => {
+    return dotty.get(composable, 'compose.' + descriptorName);
```

Comment:
Since lodash is going to split the string, it would make more sense to do:
```js
return _.get(composable, [ 'compose', descriptorName ]);
```
Also matches the format ImmutableJS uses.